mitodl/open-discussions
# Add channel header to post detail and channel settings

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1388 
Fixes #960 

#### What's this PR do?
Adds `withChannelHeader`, refactors `withChannelSidebar` and uses them on the post list and channel settings page

#### How should this be manually tested?
Click between the various pages, especially the post list and channel settings page. Only part of the page should refresh. For the channel settings page you should not see the channel header navbar, only the settings navbar.
